docs/api-reference/localization/overview.md

```diff
@@ -40,13 +40,13 @@ Let's say we want to add the Swedish language to kepler.gl. Easiest way to add t
     sv: {...en_flat, ...flattenMessages(sv)}
   };
   ```
-- Update _LOCALES_ in `src/constants/default-settings.js` to include new language:
+- Update _locales_ in `src/localization/locales.js` to include new language translation:
```

Comment:
`src/localization/index.js`

Reply:
language translations are in locales.js file